plugin/storage/cassandra/spanstore/query_builder_test.go

```diff
@@ -64,101 +65,101 @@ func TestGoodQueryScenarios(t *testing.T) {
 		{
 			&spanstore.TraceQueryParameters{NumTraces: 25},
 			expectedQueries{
-				mainQuery:       "SELECT trace_id, span_id FROM traces ALLOW FILTERING",
-				mainQueryParams: nil,
+				mainQuery:       "SELECT trace_id, span_id FROM traces LIMIT ? ALLOW FILTERING",
```

Comment:
@badiib I recall we already added limit once, but we had to roll it back because we were getting the wrong results. Do you remember the details?